.github/workflows/swift-toolchain.yml

```diff
@@ -213,7 +228,64 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        arch: ['amd64', 'arm64', 'x86']
+        include:
+          - arch: amd64
```

Comment:
Names might be nice at this point - this is a pretty diverse set of configuration.

Reply:
I set the name of the job using `name: ${{ matrix.os }} ${{ matrix.arch }} SQLite3`. Is that what you meant, or did you want to include a specific separate name in each matrix item?

Reply:
I was thinking about the matrix entry itself. But I suppose that isn't too important.